### DISCUSSION TITLE
Undo move to discussions

### DISCUSSION THREAD

**Discussion by heinrich5991** [community/community#23457] [New to GitHub]
<p>I want to undo moving some issues to discussions. I couldn’t find any way to do this in GitHub itself.</p>
<p>Re-opening the issues that were moved doesn’t work, because they’re in some special “moved to discussions” state.</p>
<p>Using the “open an issue for this discussion” doesn’t work because it doesn’t include any comments, and doesn’t even link to the original issue.</p>
<p>Is there some sort of program or bot I can use to automate this?</p>

**Comment by jfullstackdev** [comment]
<p>I think it’s permanent at this point, since there is a warning before you convert it, much like changing your username</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077199-26b460b7-cf64-40a6-a7cd-5655328269e6.png" alt="image" data-base62-sha1="5OQyUTQ2dFRagYI8jLAZ4yHnTGH" width="467" height="323"></p>

**Comment by heinrich5991** [comment, marked as answer]
<p>In the end, I was able to unlock and re-open the issues after disabling GitHub discussions.</p>
<p>I don’t know if that method was enabled by GitHub support, I also contacted them.</p>
